.azure-pipelines/specialcase.test.yml

```diff
@@ -31,6 +32,7 @@ jobs:
      }
      
      pypy3 ./setup.py check -r -s
+     Set-Item -Path Env:PATH -Value ($storedPath)
```

Comment:
Just like you got the value via `$env:PATH` you can set it in the same way like `$env:PATH=$storedPath`

Reply:
Tried. And it worked.